lib/helper/WebDriverIO.js

```diff
@@ -452,6 +452,14 @@ class WebDriverIO extends Helper {
     return findFields.call(this, locator).then((res) => res.value);
   }
 
+  static _throwNotFoundError(locator, prefixMessage = "Element",
```

Comment:
can this be handled by new NotFoundError class instead of a custom method?
Something like

```js
throw new ElementNotFound(locator, msg);
```
It would be much cleaner this way